aioimaplib/aioimaplib.py

```diff
@@ -422,8 +423,10 @@ async def execute(self, command: Command) -> Response:
                 await self.pending_async_commands[command.untagged_resp_name].wait()
             self.pending_async_commands[command.untagged_resp_name] = command
 
+        print("actually sending command")
```

Comment:
Looks like you forgot some debug `print`?

Reply:
Yes, sorry about that. Removed it.